sdk/core/core-tracing/src/utils/traceParentHeader.ts

```diff
@@ -1,63 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-import { SpanContext, TraceFlags } from "../interfaces";
-
-const VERSION = "00";
-
-/**
- * Generates a `SpanContext` given a `traceparent` header value.
- * @param traceParent - Serialized span context data as a `traceparent` header value.
- * @returns The `SpanContext` generated from the `traceparent` value.
- */
-export function extractSpanContextFromTraceParentHeader(
```

Comment:
ah this logic all lives in the instrumenter now I guess?

Reply:
Yeah, let me know what you think but I figured it makes sense to bundle all the functionality that is implementation-specific into the single instrumenter interface